vcs/vcs.go

```diff
@@ -43,7 +57,7 @@ func Nearest(dirname string) (VCS, string, error) {
 		if err != nil {
 			return err
 		}
-		if tool == Subversion || tool == Mercurial || tool == Bazaar {
+		if tool == Mercurial || tool == Bazaar {
 			return ErrUnsupportedVCS
```

Comment:
So I'm not sure what to make of this together with your comment on the use of `Nearest` in `golang/project.go`. I agree that maintaining a list of unsupported VCSs in `Nearest` is funny. Yet I was afraid to change much here because I knew it could affect how the function is used in `golang/project.go`. Can you tell me more about the error you got?

Reply:
Don't let that part scare you, it is the last of a few fallbacks used to determine the project import path. Any of the changes in this function shouldn't change the behavior in `golang/project.go`, and you shouldn't worry about changing how `golang/project.go` handles the error either. If any error is returned from `Nearest()`, then `project.go` will return the error and analysis will stop, which is expected.